tests/common/layerrenderer.cpp

```diff
@@ -35,18 +41,28 @@ bool LayerRenderer::Init(uint32_t width, uint32_t height, uint32_t format,
     return false;
   }
 
+#if USE_MINIGBM
   int gbm_bo_fd = gbm_bo_get_plane_fd(gbm_bo_, 0);
+#else
+  int gbm_bo_fd = gbm_bo_get_fd(gbm_bo_);
+#endif
   if (gbm_bo_fd == -1) {
     printf("LayerRenderer: gbm_bo_get_fd() failed\n");
     return false;
   }
 
+#if USE_MINIGBM
   planes_ = gbm_bo_get_num_planes(gbm_bo_);
   for (size_t i = 0; i < planes_; i++) {
     native_handle_.import_data.offsets[i] = gbm_bo_get_plane_offset(gbm_bo_, i);
     native_handle_.import_data.strides[i] = gbm_bo_get_plane_stride(gbm_bo_, i);
     native_handle_.import_data.fds[i] = gbm_bo_fd;
   }
+#else
+  planes_ = hwcomposer::gbm_bo_get_num_planes(gbm_bo_get_format(gbm_bo_));
+	native_handle_.import_data.stride = gbm_bo_get_stride(gbm_bo_);
```

Comment:
You can check rest of code base, everything should be aligned and that clang-format thing I mentioned above should do it for you. We need to add that to wiki